_includes/source_code/code/25-PDP/karla/karla_efficient_disjoint.cc

```diff
@@ -0,0 +1,60 @@
+#include <cstdio>
+#include <cassert>
+
+using namespace std;
```

Comment:
Χρησιμοποιείτε κάτι από το std?

Reply:
printf,scanf,freopen
για να το βγάλω θα πρέπει να κάνω include to stdio.h